.gitignore

```diff
@@ -323,3 +323,5 @@ platform/windows/godot_res.res
 # compile commands (https://clang.llvm.org/docs/JSONCompilationDatabase.html)
 compile_commands.json
 
+# CLion project files
+.idea
```

Comment:
Maybe remove line 20: `platform/android/java/.idea/*` in this case?

Reply:
I better change it to `.idea/*` than changing existing line.